tools/cfn2ts/lib/codegen.ts

```diff
@@ -232,7 +232,7 @@ export default class CodeGenerator {
     this.code.line(' *');
     this.code.line(' * @experimental');
     this.code.line(' */');
-    this.code.openBlock(`public static fromCloudFormation(scope: ${CONSTRUCT_CLASS}, id: string, resourceAttributes: any): ` +
+    this.code.openBlock(`public static fromCloudFormation(scope: ${CONSTRUCT_CLASS}, id: string, resourceAttributes: any, finder: ${CORE}.ICfnFinder): ` +
```

Comment:
Maybe it's worth sticking these in an object parameter pre-emptively, in case we need to add more parameters in the future?

Reply:
Should finder be optional?

Reply:
Given that you have added this, doesn't it make sense to start using it in resolving `{Ref}`s and `{Fn::GetAtt}`s as well?

Reply:
> Maybe it's worth sticking these in an object parameter pre-emptively, in case we need to add more parameters in the future?

Done.

Reply:
> Should finder be optional?

What should be the behavior when `finder` has not been provided?

Reply:
> Given that you have added this, doesn't it make sense to start using it in resolving `{Ref}`s and `{Fn::GetAtt}`s as well?

Of course! Not in this PR though.